playspace_app/db/migrate/20151101155610_drop_categories_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropCategoriesTable < ActiveRecord::Migration
+  def change
+    drop_table :categories
```

Comment:
if you need to redo the database, you should run `rake db:drop` and then modify your migrations and run `rake db:migrate`, rather than writing a migration specifically to delete the table